cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
```

Comment:
This is a code quality issue. The validation logic for the ticket is tightly coupled with the serializer. Consider moving the validation logic to the model or a separate service. This way, the validation logic can be reused elsewhere and the serializer's responsibility is kept to serializing and deserializing data.